app/public/locales/en/translation.json

```diff
@@ -71,6 +76,7 @@
   "total-deploys": "Total Deploys",
   "total-rows": " total rows",
   "today": "Today",
+  "total-stake": "Total Stake",
```

Comment:
Any idea when we'll get german translations for BE?

Reply:
I don't. Maybe @hoffmannjan has some idea?